amical/mf_pipeline/ami_function.py

```diff
@@ -385,6 +385,12 @@ def make_mf(
     filt = get_wavelength(instrument, filtname)
 
     if instrument == "SPHERE-IFS":
+        if i_wl is None:
+            raise ValueError(
+                "Your file seems to be obtained with IFU instrument: spectral "
+                + f"channel index `i_wl` must be specified (nlambda = {len(filt)}) and "
+                + "should be strictly identical to the one used for the cleaning step."
```

Comment:
```suggestion
                f"channel index `i_wl` must be specified (nlambda = {len(filt)}) and "
                "should be strictly identical to the one used for the cleaning step."
```

Reply:
More importantly, I don't understand this message. It's raised on a condition that depends only on parameters the function receives directly, but it's pointing at a problem somewhere else ?

Reply:
Yes, you need to use the same `i_wl` argument in both function (clean and extract). That's why I added this message but it's maybe useless to detail this point there.